README.md

```diff
@@ -120,7 +120,7 @@ To use the app from the repository root:
 
 ### 2. Launch the app
 
-To build the Docker images from the repository Dockerfiles, run the following command:
+To build the Docker images from the head of the repository, run the following command:
```

Comment:
modification declined for the following reasons:
- line 115 already specifies that these instructions refer to the use of the app **from the repository root**
- the specific goal of line 123 is to explain the first method of sourcing the docker images (the second method is explained by line 129)